modules/Grocery-Queries/README.md

```diff
@@ -1,10 +1,13 @@
 # Grocery Queries Exercise
 
-___Note:__ This module assumes you've already installed postgresql via homebrew. If you have not done that please do back and do the [Installfest](../Installfest) module._
+___Note:__ This module assumes you've already installed postgresql via homebrew. If you
```

Comment:
these changes should not be in this PR. please remove

Reply:
Very curious that our README somehow picked up those changes. Not even sure what or whose branch that is from. Will revert... just curiously commenting.

Reply:
> Very curious that our README somehow picked up those changes.
Well technically you committed these changes and worse you submitted them for reviews. This is the perfect time for you to focus on leveling up your git hygiene. Get into the habit of always using `git add -p` and before you open a PR always: rebase off upstream/master and consider squashing and polishing your commits. double or even triple check the diff.